```
# github.com/gravitational/teleport/lib/events
lib/events/uploader.go:222:26: too many arguments in call to os.Open
	have (string, int, number)
	want (string)
# github.com/gravitational/teleport/lib/events [github.com/gravitational/teleport/lib/events.test]
lib/events/uploader.go:222: too many arguments in call to os.Open
	have (string, int, number)
	want (string)
FAIL	github.com/gravitational/teleport/e/lib/auth [build failed]
```

--- a/lib/events/uploader.go
+++ b/lib/events/uploader.go
@@ -219,7 +219,7 @@ func (u *Uploader) emitEvent(e UploadEvent) {
 }
 
 func (u *Uploader) uploadFile(lockFilePath string, sessionID session.ID) error {
-	lockFile, err := os.Open(lockFilePath)
+	lockFile, err := os.OpenFile(lockFilePath, os.O_RDWR, 0)
 	if err != nil {
 		return trace.ConvertSystemError(err)
 	}

